scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -64,19 +70,20 @@ But for your own reference, here is how you'd do it yourself:
 #  -k 23
```

Comment:
Should we change the `-i` bit to `-i index/Mus_musculus/short_index` to match the paths the participants will have?